test_util/test_runner.py

```diff
@@ -1,28 +1,52 @@
 #!/usr/bin/env python3
 """Module for running integration_test.py inside of a remote cluster
-Parameters for integration_test.py are passed via the same environment variables
 Note: ssh_user must be able to use docker without sudo priveleges
 """
 import logging
+import shutil
+import tempfile
 import time
 from contextlib import contextmanager
 from multiprocessing import Process
 from os.path import join
-from subprocess import CalledProcessError, TimeoutExpired
+from subprocess import CalledProcessError, TimeoutExpired, check_call
 
 import pkg_resources
 
 LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
 logging.basicConfig(format=LOGGING_FORMAT, level=logging.DEBUG)
 log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+TEST_DOCKERFILE = """FROM py.test:noop
+ENV DCOS_DNS_ADDRESS=http://{dcos_dns}
```

Comment:
Every one of these ENV lines adds another docker layer, and each layer adds a bit of time.  Would be better to do a "wrapper file" that is copied in which sets all of these. See: https://github.com/dcos/dcos/blob/master/gen/installer/bash/installer_internal_wrapper.in for what we do in dcos_generate_config.sh's docker.